ansible/playbook.yml

```diff
@@ -93,9 +93,23 @@
   roles:
     - cyhy_ops
 
+- hosts: cyhy_feeds
```

Comment:
It looks like this block has been duplicated; also, this block has an incorrect name.